ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py

```diff
@@ -164,3 +164,30 @@ def test_paddle_acosh(
         atol=1e-2,
         x=x[0],
     )
+
+
+# asin
+@handle_frontend_test(
+    fn_tree="paddle.tensor.math.asin",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
```suggestion
        available_dtypes=helpers.get_dtypes("valid"),
```

Can you make this change and let me know if you face any issues?

Reply:
@hello-fri-end ,I made the changes , now two test failing intelligent-tests-pr / run_tests (1 and 6)